Source/Details/ASTraitCollection.h

```diff
@@ -48,6 +48,10 @@ typedef struct {
   unowned UIContentSizeCategory preferredContentSizeCategory API_AVAILABLE(ios(10.0));
 
   CGSize containerSize;
+  
```

Comment:
Do these changes compile in Xcode 10? We should probably make this work in both the Xcode 11 beta and Xcode 10 GM until we have a GM version of Xcode 11